cocassel/milk-buddy
# Updated scrolling on container page

**Testing Instructions**
1. Navigate to the container page 
2. Scroll to the bottom of the page

You should see the entire "Next" button. It should not be cut off. Test on smaller phones
